pkg/daemon/rpm-ostree.go

```diff
@@ -77,6 +78,22 @@ func NewNodeUpdaterClient() NodeUpdaterClient {
 	return &RpmOstreeClient{}
 }
 
+// Synchronously invoke rpm-ostree, writing its stdout to our stdout,
+// and gathering stderr into a buffer which will be returned in err
+// in case of error.
+func runRpmOstree(args ...string) error {
```

Comment:
Perhaps we can rename it to something more generic like `runCmd(command string, args ...string)` or something similar. We are using runGetOut() function today for running non rpm-ostree command as well like https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/update.go#L105 .

Reply:
I looked at this and updated the commit message around this too.  In the systemctl case...it doesn't really print to stdout, but the extra logging we do makes sense I think for this one.  So updated!

Reply:
(I kept the `runRpmOstree()` thin wrapper because we call it often and we may want to do something else there in the future)